### ISSUE TITLE
New Rule: no-duplicate-keyframe-selectors

### ISSUE THREAD

**Issue by snitin315** [eslint/css#141]
### Rule details

Disallow duplicate selectors within keyframe blocks.

### What type of rule is this?

Warns about a potential problem

### Example code

```css
/* eslint css/no-duplicate-keyframe-selectors: "error" */

@keyframes foo { 0% {} 0% {} } // error

@keyframes foo { from {} from {} } // error

@keyframes foo { 0% {} 100% {} } // ✅

@keyframes foo { from {} to {} } // ✅
```

### Participation

- [x] I am willing to submit a pull request to implement this rule.

### Additional comments

corresponding stylelint rule - https://stylelint.io/user-guide/rules/keyframe-block-no-duplicate-selectors

**Comment by nzakas** [comment]
Sounds like a good idea. :+1: Marking as accepted.

**Comment by snitin315** [comment]
@thecalamiity Thanks, but I have already started working on it.
